packages/dev-scripts/examples/util.ts

```diff
@@ -162,9 +168,8 @@ export function getExampleProjects(): Project[] {
         .split(path.sep);
 
       const group = {
-        pathFromRoot: path.relative(
-          path.resolve("../../"),
-          path.join(directory, "..")
+        pathFromRoot: slash(
```

Comment:
I'm not sure `slash` is needed (would prefer not to add this dependency). maybe we can just prevent passing / to path.resolve? e.g.: `path.resolve("..","..")`

Reply:
Yes, I'd love to see this code a bit clearer / more uniform. When reviewing, it's not entirely clear to me why we have 3 different methods:
- `replace`
- `slash`
- `convertPathToPattern`

Can we either use 1 method (ideally with a separate, well named helper function). Or, if multiple methods are necessary, add a comment in code why we use each function?